src/app/tabs/settings-tab.js

```diff
@@ -115,7 +115,7 @@ module.exports = class SettingsTab {
         </div>
         <div class="${css.crow}">
           ${self._view.theme.david}
```

Comment:
is it also possible to rename underlying property/variable/filename ? (just to avoid confusion)

Reply:
fixing errors and pushing

Reply:
https://github.com/JeremiGendron/remix-ide/tree/patch-03 can't resolve this: changed all occurrences of "david" to clean with proper casing and renamed styleGuideDavid to styleGuideClean, still getting runtime errors.

I'm going to bed now, I am sad :(

Reply:
yes but the code still contain `david` :)
could you add your commit https://github.com/JeremiGendron/remix-ide/commit/346ad992afce67f2550ab42838538d5f5b7a4580
to this PR? `git cherry-pick <commit-id>` should work
I can take a closer look too.
thanks